### PR TITLE
updated news.md and readme.md for 1.8.0 release

### PR DESCRIPTION
**Pull request by samskalicky**
## Description ##
updated news.md and readme.md for 1.8.0 release

### CODE REVIEW

**Comment by ciyongch** [comment]
It's great to see 1.8.0 release, kudos to @samskalicky ! 
The original template of the news for major/minor release is kind of trivial :-P.
Is this the new format for the upcoming release as all the changes items are in a single bullet, is it better to keep at least new features (like intgemm/cudagraph) and feature improvements? Which will be easier to get the newly introduced features.

**Comment by samskalicky** [comment]
> It's great to see 1.8.0 release, kudos to @samskalicky !
> The original template of the news for major/minor release is kind of trivial :-P.
> Is this the new format for the upcoming release as all the changes items are in a single bullet, is it better to keep at least new features (like intgemm/cudagraph) and feature improvements? Which will be easier to get the newly introduced features.

Honestly, I dont think we need this News.md updated as we have been doing. Its pretty much a duplication of whats in the release notes: https://github.com/apache/incubator-mxnet/releases and in the Readme.md we actually link to that page. The News.md isnt used anywhere anymore, users actually have to go click it to view it. it doesnt show up on the main GitHub page or the website anymore.

File: NEWS.md [hunk 1]
```diff
@@ -274,6 +290,114 @@ MXNet Change Log
...
+ - Add new CI pipeline for building and testing with cuda 11.0. (#19149)
+ - Enable CUDA 11.0 on nightly development builds (#19314)
+#### TensorRT
+ - TensorRT: add int8 with calibration (#19011)
+ - Add TRT verbose mode (#19100)
+ - Backporting TensorRT-Gluon Partition API (and TensorRT 7 support) (#18916)
+ - Backport TRT test update #19296 (#19298)
+#### OneDNN
+ - Upgrade to oneDNN v1.6.3 (#19153) (#19161)
+ - Update oneDNN to official v1.6 release (#18867) (#18867)
```
Comment (ciyongch) [inline NEWS.md:310]: duplicated PR number?

**Review by ciyongch — APPROVED** [review]
LGTM

**Pull request merged**